pai-management/doc/cluster-bootup.md

```diff
@@ -337,9 +338,10 @@ machines:
   - 192.168.1.11
 
 # (Required) Log-in info of all machines. System administrator should guarantee
-# that the username/password pair is valid and has sudo privilege.
+# that the username/password pair or username/key-filename is valid and has sudo privilege.
 ssh-username: pai
 ssh-password: pai-password
+ssh-key-filename: key-filename
```

Comment:
Should be a path to the keyfile？

Reply:
Yes, it is a path to keyfile, Maybe better to name it to  keyfile-path?

Reply:
Yes, ```ssh-keyfile-path``` make sense.